backend/danswer/db/slack_bot_config.py

```diff
@@ -79,12 +80,23 @@ def insert_slack_bot_config(
     persona_id: int | None,
     channel_config: ChannelConfig,
     response_type: SlackBotResponseType,
+    standard_answer_categories: list[int],
```

Comment:
Slight preference to have the variables here and elsewhere called whatever_object_id when it's not the object itself but instead the id. I think even if it's the primary key, it's still nice to have the clarification for fast reading without having to think about it

Reply:
I agree that we should standardize on this - updated ✅